snap/snapcraft.yaml

```diff
@@ -1,6 +1,6 @@
 name: charmed-mysql # you probably want to 'snapcraft register <name>'
 base: core22 # the base snap is the execution environment for this snap
-version: '8.0.36' # just for humans, typically '1.2+git' or '1.3.2'
+version: '8.0.37' # just for humans, typically '1.2+git' or '1.3.2'
```

Comment:
Consider to remove the comments here and above it the future. It was relevant example for the snap cration from template phase. :-D